Sources/Purchasing/Purchases/PurchasesType.swift

```diff
@@ -907,6 +907,20 @@ public protocol PurchasesType: AnyObject {
     @available(iOS 13.0, macOS 10.15, tvOS 13.0, watchOS 6.2, *)
     func syncAttributesAndOfferingsIfNeeded() async throws -> Offerings?
 
+    /**
+     * Redeems a web purchase previously parsed from a deep link with ``Purchases/parseAsWebPurchaseRedemption(_:)``.
+     * We recommend using ``Purchases/redeemWebPurchase(_:)`` for a nicer API unless you're using ObjC.
```

Comment:
I don't think this is a thing we should even bring up - there are valid reasons to not use an async / await API, even now that Swift Concurrency is backported to iOS 13. 

Specifically, the Swift Concurrency model and the old concurrency model don't necessarily get along all that well, so if you're using completion blocks, it might be a good idea to stick with that. 

If folks are looking for async / await, they'll find it. And those are the APIs that we'll be advertising and showing first and foremost in our docs.

Reply:
Instead, we should add it as a seealso so that folks who find this find the other version too